gnucash/python/init.py

```diff
@@ -1,15 +1,22 @@
 import sys
 import gnucash._sw_app_utils as _sw_app_utils
-from gnucash import *
+import gnucash._sw_core_utils as _sw_core_utils
```

Comment:
true, line 2 as well

Reply:
I put that in to make more obvious what is being imported. I had a look into the python style guide:

> Wildcard imports (from <module> import *) should be avoided, as they make it unclear which names are present in the namespace, confusing both readers and many automated tools. There is one defensible use case for a wildcard import, which is to republish an internal interface as part of a public API (for example, overwriting a pure Python implementation of an interface with the definitions from an optional accelerator module and exactly which definitions will be overwritten isn’t known in advance).

source https://pep8.org/#imports

I'd prefer to leave the redundancy and later on try to remove the "import *"

Reply:
This isn't the case that paragraph is talking about: It's about importing individual symbols like variables and function names. `gnucash` doesn't (or at least shouldn't) have any of those. It has only more namespaces so `from gnucash import *` means that can say `engine.xaccTransCommitEdit` instead of the `gnucash.engine.xaccTransCommitEdit` that `import gnucash` would require and doesn't require explicitly importing every module by name. Since there's still a module prefix it doesn't pollute the global namespace any more than would explicitly importing `gnucash.engine`.

 FWIW
```
from gnucash import _sw_app_utils
```
and 
```
import gnucash._sw_app_utils as _sw_app_utils
```
do the same thing and the former is more idiomatic (or as the python folks like to say, "pythonic").
I also suggest ditching

Reply:
I'm not convinced yet and will look further into this topic. Looking areound I found rather similar views that dislike import *.  An exemplary blog about it: https://medium.com/@s16h/importing-star-in-python-88fe9e8bd4d2 

Except from the more general thoughts about * imports there seems to be another thing: things starting with an underscores don't get imported. That will probably prevent _sw_app_utils and _sw_core_utils from being imported and may have led to the existing redundancy of _sw_app_utils getting imported beside *.

Reply:
I just tested if _sw_app_utils and _sw_core_utils get imported. They don't if `from gnucash import *` and need to be imported explicitly.
I wondered if the _-method would get imported. That actually works.